infinitecsolutions/restful-api-guidelines
# Initial changes

First changes, such as changing company name, adding boilerplate required by CC-BY 4.0 and removing Google Analytics.

We need to have a license statement and refer to the original content (the Zalando Guidelines). To make it possible to link to a local copy of the the actual CC license, the build script copies the LICENSE file to the output.

We do have no current interest in tracking what is viewed by readers of this document. In any case, reporting usage to a Zalando GA account needs to be removed, anyway.

#### How to verify

1. Check out this branch
2. Run `./build.sh` (Docker needs to be running!)
3. Open `output/index.html` in your browser and see that the top of the page has "infinitec" instead of Zalando and a box with language about the license.
4. Check that the new links work (to Zalando API Guidelines and to the local CC-BY license file)
